docs/reference/search/explain.asciidoc

```diff
@@ -118,15 +112,13 @@ This will yield the same result as the previous request.
 `_source`::
 
     Set to `true` to retrieve the `_source` of the document explained. You can also
-    retrieve part of the document by using `_source_include` & `_source_exclude` (see <<get-source-filtering,Get API>> for more details)
+    retrieve part of the document using `_source_include` & `_source_exclude` (see <<get-source-filtering,Get API>> for more details)
 
```

Comment:
Instead of "document explained", I'd probably say:

Set to `true` to retrieve the document `_source` along with the explanation. To retrieve a subset of the document `_source`, you can use `_source_include` and `_source_exclude`. For more information, see the <<get-source-filtering,Get API>>.